src/Admin.php

```diff
@@ -39,6 +39,10 @@ public static function init() {
     // Exclude subscribers from post author select options to prevent a performance
     // slowdown on sites with large amounts of non-administrative registered users.
     add_filter('wp_dropdown_users_args', __CLASS__ . '::wp_dropdown_users_args');
+
+    // Copies .htaccess template file into uploads directory.
+    add_action('wp_upgrade', __CLASS__ . '::wp_upgrade', 10, 2);
+    add_action('wp_install', __CLASS__ . '::wp_install');
```

Comment:
1. `wp_install` is fired when a new site is fully installed.  At that point, this plugin will not exist yet, so registering this filter is pointless.

1. You don't need to create and register a separate callback function if all arguments are ignored and all it does is to forward to another function.  You can register the final function instead.

1. The callback function for this special operation should be located in the Schema class.

1. These special actions should be registered from the main plugin file.